atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockIntegrationTest.java

```diff
@@ -219,22 +214,7 @@ private SchemaMutationLock createQuickHeartbeatTimeoutLock() {
     }
 
     private void setUpWithNonHeartbeatClearedLock() throws TException {
```

Comment:
Can you give this a better name while you're here? This name only makes sense if you have worked on the heartbeat stuff before it was modified. Something like `setLockToClearedOldStyle`